code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -216,7 +216,20 @@
 	key_third_person = "salutes"
 	message = "salutes."
 	message_param = "salutes to %t."
+	emote_type = EMOTE_SOUND | EMOTE_VISIBLE
```

Comment:
```suggestion
```
Let it default to EMOTE_VISIBLE Henri

Reply:
Oh thank you Henri, you are so smart!

Reply:
Truly the smartest reviewer of the entire codebase